controller/partyController.js

```diff
@@ -0,0 +1,52 @@
+import db from "../db/db"
+
+class PartyController{
+    createParty(req, res){
+
+        const body = req.body;
+        const {name, hqAddress, logoUrl} = body;
+        const id = db.length + 1;
+        const party = {
+            id,
+            name,
+            hqAddress,
+            logoUrl
+        }
+
+        db.push(party);
+
+        res.status(201).send({
+            status: 201,
+            data: [{
+                id,
+                name
+            }]
+        })
+    }
+
+    getAllParty(req, res){
```

Comment:
You should get the parties from your database.

Reply:
I think we are not expected to use database until challenge 3. please kindly confirm that. I was getting the parties from the data created in db.js

Reply:
You have done that already in this P.R #46